nservicebus/messaging/send-a-message.md

```diff
@@ -12,6 +12,41 @@ related:
 
 NServiceBus supports sending different types of messages (see [Messages, Events, and Commands](messages-events-commands.md)) to any endpoint. Messages can be sent either directly from the endpoint or as part of handling an incoming message. When a message arrives at an endpoint, it goes through a [pipeline of processing steps](/nservicebus/pipeline/).
 
+## Message sending/publishing interface summary
+
+| Feature      | IMessageSessionContext | IEndpointInstance | IMessageSession |
+|--------------|------------------------|-------------------|-----------------|
+| Can send/publish/reply to messages          | ✅  | ✅ | ✅ |
+| Use inside a handler          | ✅ |  |  |
+| Use outside of handler |  | ✅ | ✅ |
+| Takes part in message handler transaction   | ✅ |  |  |
+| Auto-injected into dependency injection container         |  |  | ✅ |
+| Access to endpoint lifecycle control          |  | ✅ |  |
+|||||
```

Comment:
Is this empty row needed?

Reply:
It's to show a line underneath the table to kind of finish it off.